packages/config/index.js

```diff
@@ -40,14 +74,13 @@ function resolvePaths(config) {
         }
       })(config[key]);
     });
-  return config;
+  return result;
 }
 
 function normalizeURLs(config) {
   var basePath = config.basePath.replace(/^\/*/, '/').replace(/\/*$/, '');
```

Comment:
Could we change this so that the `basePath` *could* contain a trailing slash (if one is present in the user config) and only trim it to a single slash (if present), otherwise continue without a trailing slash?

Reply:
While we certainly could, I do not really see any benefit, here - but that would imply touching a whole lot of different places (react-router config springs to mind).

Reply:
Ok, let's keep it out of this PR and I'll think about it a bit more and create an issue for this.